proxy/proxy_test.go

```diff
@@ -2268,6 +2268,286 @@ func TestProxyFromEmptyContext(t *testing.T) {
 	assert.Nil(t, proxyUrl)
 }
 
+const (
```

Comment:
All moved from loadbalancer/fadein_test.go

Reply:
Can we have a new _test.go file for this. proxy_test.go is quite huge already

Reply:
Yes, will do.